week2/task7/Program.cs

```diff
@@ -0,0 +1,103 @@
+﻿var signaler = new Signaler();
+signaler.AddTime(new JupiterTime(2, 00));
+signaler.AddTime(new JupiterTime(4, 00));
+signaler.AddTime(new JupiterTime(6, 00));
+signaler.Check(new JupiterTime(4, 21));
+
+class JupiterTime
+{
+    public int _hours = 0, _minutes = 0;
```

Comment:
These should be set to private, otherwise they can be accessed from outside the 'JupiterTime' class and that can bring unexpected behavior